dev/com.ibm.ws.security.jaas.common/src/com/ibm/ws/security/jaas/common/internal/JAASLoginModuleConfigImpl.java

```diff
@@ -407,7 +406,7 @@ protected void setClassLoadingSvc(ClassLoadingService classLoadingService) {
     }
 
     private static boolean isIBMJdk18() {
-        return (JavaInfo.vendor() == Vendor.IBM && JavaInfo.majorVersion() == 8);
+        return JavaInfo.isAvailable("com.ibm.security.auth.module.Krb5LoginModule");
     }
```

Comment:
The name of this method should probably change to something more descriptive.

Reply:
I think the method can go away entirely ... under a separate PR ;-)

Reply:
https://github.com/OpenLiberty/open-liberty/issues/18055

Reply:
https://github.com/OpenLiberty/open-liberty/pull/18053